spec/jasmine-helper.js

```diff
@@ -0,0 +1,222 @@
+'use babel'
+/* @flow */
```

Comment:
Can you use a `//` comment for the flow directive? If so, we could do this:

```js
// @babel
// @flow
```

If not, maybe we should add `/* @babel */` as another possible prefix for es6 files in Atom.